File: Frontend/betacycle/src/app/shared/models/address.ts
import { User } from "./user";

export class Address
{
    userId: number = 0;
    city: string = '';
    address: string = '';
    cap: string = '';
    province: string = '';
    nation: string = '';
    addressID: number = 0;
    user: User = new User();
}

File: Betacycle BackEnd Aggiornato 20-05/BetaCycle/Models/Address.cs
using System;
using System.Collections.Generic;

namespace BetaCycle.Models;

public partial class Address
{
    public long UserId { get; set; }

    public string City { get; set; } = null!;

    public string Address1 { get; set; } = null!;

    public string Cap { get; set; } = null!;

    public string Province { get; set; } = null!;

    public string Nation { get; set; } = null!;

    public long AddressId { get; set; }

    public virtual User User { get; set; } = null!;
}
